.travis.yml

```diff
@@ -21,21 +22,22 @@ matrix:
 
     # OSX 10.10 Yosemite
     - osx_image: xcode6.4
+    # macOS 10.13 High Sierra
+    - osx_image: xcode10.1
     # macOS 10.12 Sierra
     - osx_image: xcode9.2
     - osx_image: xcode8.3
     # OSX 10.11 El Capitan
-    - osx_image: xcode7.3
+    - osx_image: xcode8
```

Comment:
Huh. I originally removed our `xcode8` build because Travis announced that they would deprecate that image (2b568ae4decb0b342dcbcce49009b58cae5b9ad0). But apparently they un-deprecated `xcode8` and changed their blog post without any comment, so my commit message now looks like nonsense... (I'm not making this up - compare the blog post [on 2017-11-27](https://web.archive.org/web/20171127041023/https://blog.travis-ci.com/2017-11-21-xcode8-3-default-image-announce#retiring-some-older-images) and [on 2018-06-14](https://web.archive.org/web/20180614162535/https://blog.travis-ci.com/2017-11-21-xcode8-3-default-image-announce#retiring-some-older-images).)

In that case re-adding `xcode8` is not a bad idea, but I think we should also keep `xcode7.3`, so that we have at least one build per major Xcode version.

Reply:
xcode8 seems buggy for our purposes because it does not contain pipenv.